libs/core/src/components/sage-input/sage-input.scss

```diff
@@ -111,7 +109,7 @@ input {
     color: var(--color-placeholder);
   }
 
-  &:has(~.sage-input__error-text) {
+  &:has(~ .sage-input__error-text) {
```

Comment:
Just a note: This css doesn't work in FF yet but it is coming. It works in Firefox with flags enabled. Just something to keep an eye on for now I think.